content/consultingv2/net-8.json

```
@@ -0,0 +1,421 @@
+          }
+        },
+        {
+          "guid": "s22wwo",
+          "embed": "Ft2JArXEXcI?si=FnoENuD0MO8JIQMJ",
+          "image": "/images/Louis Test Images/Incident-io-batch/dotnet_cleanarchitecture.png",
+          "altText": "Full-Stack Clean Architecture: The Right Way to Share Code",
+          "chips": {
+            "filledChipText": ".NET User Groups",
+            "clearChipText": "1 hour"
+          },
+          "heading": "Full-Stack Clean Architecture: The Right Way to Share Code",
+          "description": "Stop wrestling with UI and API code! Matt Goldman (author of .NET MAUI in Action) reveals how to nail Clean Architecture across MAUI and ASP.NET Core without the classic pitfalls. Expect real demos, not just theory.",
+          "embeddedButton": {
+            "buttonText": ""
+          }
+        }
+      ],
+      "background": {
+        "backgroundColour": 5
+      },
+      "_template": "cardCarousel"
+    },
+    {
+      "heading": "Frequently Asked Questions",
+      "body": "Need quick answers about .NET 8 migration? Here's what tech teams commonly ask us during assessment calls.",
+      "accordionItems": [
+        {
+          "label": "Is .NET 8 worth the migration effort?",
+          "content": "Yes, if you're running applications on older .NET versions. .NET 8 offers:\n\n* Extended support until November 2026\n* Significant performance improvements\n* Better cloud integration\n* Latest security features and patches\n* Access to new C# 12 features\n"
+        },
+        {
+          "label": "What about our existing dependencies?",
+          "content": "We audit your NuGet packages first. Most popular packages already support .NET 8. For others, we'll either:\n\n* Update to compatible versions\n* Find modern alternatives\n* Create compatibility shims\n"
+        },
+        {
+          "label": "How do we prep our team for .NET8?",
+          "content": "Quick checklist:\n\n1. Set up .NET 8 SDK\n2. Review breaking changes\n3. Run compatibility analyzer\n4. Test on a small project\n5. Plan deployment strategy\n"
+        },
+        {
+          "label": "What kind of support is available post-migration?",
+          "content": "Our support includes:\n\n* Performance monitoring\n* Security updates\n* Optional managed services\n* Regular health checks\n* Continuous optimization\n"
+        }
+      ],
+      "isMultipleOpen": true,
+      "background": {
+        "backgroundColour": 5
+      },
+      "_template": "accordionBlock"
+    },
+    {
+      "technologyGroups": [
+        {
+          "technologyGroup": "content/consultingv2Technologies/groups/Some-other-tech.mdx"
+        }
+      ],
+      "techCardStyle": 0,
+      "background": {
+        "backgroundColour": 5
+      },
+      "_template": "technologyCardCarousel"
+    },
+    {
+      "topLabel": {
+        "labelText": ""
+      },
+      "heading": "Talk to us about your  project today",
+      "isH1": false,
```

```suggestion
      "heading": "Talk to us about your .NET 8 project today",
      "isH1": false,
```

On the other consulting pages we mention the project type